.github/workflows/basis.yml

```diff
@@ -30,7 +30,7 @@ jobs:
         uses: actions/cache@v2
         with:
           path: .venv
-          key: venv-${{ runner.os }}-${{ matrix.python-version }}-${{ hashFiles('**/pyproject.toml') }}
+          key: venv-${{ runner.os }}-${{ matrix.python-version }}-${{ hashFiles('**/poetry.lock') }}
```

Comment:
We don't check in `poetry.lock` file, so we should leave this as-is.

Reply:
Ah right... forgot this one is different than others